### ISSUE TITLE
No keyboard access to chat menus in the chat history

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3788]
**Issue Description**: Each chat in the chat history has a pop-up menu to rename, share, delete, or archive, but the menu button is not accessible unless the chat thread is selected. Navigating to the chat's menu button can be difficult and unclear.

**WCAG Criteria**: [2.1.1 Keyboard - A](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

Screenshot or Screen Recording: [Screen Recording](https://share.zight.com/yAueWr8b)

**Notes**:
Have the chat link respond to focus in the same way it does on hover so that the menu is available. When it is visible, the menu is accessible; it just needs to be exposed on focus the same as it is on hover. [Example Video](https://share.zight.com/9ZuJPdXG)

<img width="452" alt="Screenshot 2024-08-26 at 4 22 01 PM" src="https://github.com/user-attachments/assets/3dde8e29-3b14-42a4-96b0-cfb955bf2bd8">
